src/modlistsortproxy.cpp

```diff
@@ -177,22 +177,7 @@ bool ModListSortProxy::lessThan(const QModelIndex &left,
       }
     } break;
     case ModList::COL_CONTENT: {
-      std::vector<ModInfo::EContent> lContent = leftMod->getContents();
-      std::vector<ModInfo::EContent> rContent = rightMod->getContents();
-      if (lContent.size() != rContent.size()) {
-        lt = lContent.size() < rContent.size();
-      }
-
-      int lValue = 0;
-      int rValue = 0;
-      for (ModInfo::EContent content : lContent) {
-        lValue += 2 << (unsigned int)content;
-      }
-      for (ModInfo::EContent content : rContent) {
-        rValue += 2 << (unsigned int)content;
-      }
-
-      lt = lValue < rValue;
+      lt = leftMod->getContents() < rightMod->getContents();

```

Comment:
How does this work?

Reply:
It's lexical comparison of the two vectors. But I noticed it's not equivalent to the old way, I need to change this.